data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/controller/OrganizationServiceController.java

```diff
@@ -0,0 +1,51 @@
+package gov.cdc.etldatapipeline.organization.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import gov.cdc.etldatapipeline.organization.config.KafkaConfig;
+import gov.cdc.etldatapipeline.organization.config.KafkaStreamsConfig;
+import gov.cdc.etldatapipeline.organization.service.OrganizationStatusService;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController
+public class OrganizationServiceController {
+
+    private OrganizationStatusService organizationStatusService;
+
+    @Autowired
+    private KafkaStreamsConfig kafkaStreamsConfig;
+
+    @Autowired
+    private KafkaConfig kafkaConfig;
+
+    public OrganizationServiceController(OrganizationStatusService organizationStatusSvc) {
+        this.organizationStatusService = organizationStatusSvc;
+    }
+
+
+    @GetMapping("/status")
+    @ResponseBody
+    public ResponseEntity<String> getDataPipelineStatusHealth() {
+        return this.organizationStatusService.getHealthStatus();
+    }
+
+
+    @PostMapping(value = "/organization", consumes = "application/json", produces = "application/json")
+    @ResponseBody
+    public ResponseEntity<String> postOrganization(@RequestBody String payLoad) throws JsonProcessingException {
+        KafkaProducer<String, JsonNode> producer = new KafkaProducer<>(
+                kafkaStreamsConfig.kStreamsConfigs().asProperties());
```

Comment:
surround this with a try catch block and return failed request status if there is any exception.

Reply:
Yes will do.